src/javascript/app_2/Stores/Modules/Contract/contract-store.js

```diff
@@ -73,8 +74,10 @@ export default class ContractStore extends BaseStore {
     onLoadContract(contract_info) {
         if (contract_info === this.contract_id || !contract_info) return;
         this.onSwitchAccount(this.accountSwitcherListener.bind(null));
-        this.root_store.modules.trade.symbol = contract_info.underlying;
         this.smart_chart   = this.root_store.modules.smart_chart;
```

Comment:
now that it's separated from the assignments below it, it has extra space before `=`